lua/lazyvim/plugins/extras/ui/codewindow-nvim.lua

```diff
@@ -0,0 +1,39 @@
+return {
+  {
+    "gorbit99/codewindow.nvim",
+    keys = {
+      {
+        "<leader>um",
+        function()
+          require("codewindow").toggle_minimap()
+        end,
+        desc = "Toggle minimap",
+      },
+    },
+    init = function()
+      require("codewindow")
+    end,
```

Comment:
I don't believe these lines are needed.

Reply:
Should've made a code suggestion. Sorry about that.

Reply:
I tested your changes and it stops working.

Reply:
Are you sure? Because I've also tested locally your PR and it works just fine.

Reply:
ok I see: it stops auto-opening but it is there. It bothers me having to manually open it when the setup should really do that for me. (opts auto enable). not sure what to think of this.

Reply:
You just press `<leader>um` and it opens up.

Reply:
that is exactly what I mean. I want it to open automagically for me without using the shortcut. auto enable should have that behavior but it doesn't work unless I add the init code.

Reply:
That happens because the plugin isn't loaded, but gets lazy-loaded on the press of keymap. You could put `lazy = false`, if you want that behavior. But LazyVim lazy-loads most of the prebuilt plugins. From `~/.config/nvim/lua/config/lazy.lua`
>     -- By default, only LazyVim plugins will be lazy-loaded. Your custom plugins will load during startup.

So, I believe we should adhere to LazyVim's default principles and let the user change that setting in his personal configuration if he likes. 

But I'm not the one who makes decisions, so let's just wait on the maintainer's input regarding this.

Reply:
But if you want that behavior, I'd suggest to replace those lines with `lazy = false` instead. It aligns better with how you set up `lazy.nvim`.

Reply:
I am going ahead and replacing those lines with lazy=false and see what maintainers think about this. I generally agree, but hope this gets approved. (unlikely).

Reply:
I don't mind either way. I was just stating how LazyVim operates regarding its prebuilt plugins. Please don't take my suggestion personally.

Reply:
no offense or taking personally here :) I perfectly understand your intentions and appreciate the clarification. I agree with you on all you said :) (but still hope to have one less plugin to customize think about).